### ISSUE TITLE
Code is out of date

### ISSUE THREAD

**Issue by jbhaywood** [reactiveui/rxui-design-guidelines#15]
As of ReactiveUI v5.0, ReactiveAsyncCommand is no longer available.  I'm brand new to ReactiveUI and looking for best practices before I develop a lot of bad habits.  The principles here are probably still sound, but I can't implement the code as written, and can't be totally sure some that there aren't new best practices for the newer versions.

**Comment by shiftkey** [comment]
:thumbsup: if you're interested in having a go at updating it I'm happy to review.

**Comment by jbhaywood00** [comment]
Considering I haven't actually built anything with it yet, I don't think I'm qualified.  But I'm always up for a challenge, so let me get my head around RxUI and build something useful and I'll give it a go.

Given that this is the #1 Google hit for "reactiveui best practices", as well as being referenced from other blogs I've come across, it would be great to see it maintained.

It's a shame though that RxUI docs in general seem to be in a sorry state.  It makes me wonder if I should be spending my "personal project" time learning it.  Even the official docs contain some pages that are nothing more than Twitter conversations.  Is this a dead framework?  It definitely looks appealing and worth investing some time in, so I hope not.

**Comment by shiftkey** [comment]
Have you seen http://reactiveui.readthedocs.org/?

**Comment by jbhaywood00** [comment]
I sure have.  It starts out great, but then you get to Binding and Routing and it turns into a bunch of chat logs.  It took me a minute to just figure out what I was looking at.  Those sections are hard to follow overall and give the impression that someone was in a hurry and then couldn't be bothered to go back and clean it up.

Don't get me wrong, I've been working with traditional WPF/MVVM and Observables for years and I'm excited to put the two together and dive into RxUI, especially because of cross-platforms implications.  But the feeling I get as a newcomer is that marketing, evangelizing, and making it easy for new users to get up to speed isn't high on the priority list.

**Comment by shiftkey** [comment]
Yeah, it's an ongoing task and unfortunately documentation in general isn't glamorous work

**Comment by jbhaywood00** [comment]
So you became a programmer for the glamorous lifestyle?  I fear you may have been misinformed by your guidance counselor. :)

**Comment by ghuntley** [comment]
> I get as a newcomer is that marketing, evangelizing, and making it easy for new users to get up to speed isn't high on the priority list.

@jbhaywood It's actually very high on the priority list, pretty much my sole task right now. Since November here's what's happened on the marketing front.

* All existing issues have been properly categorized/labeled.
* Rewrote the main repository README.md so that is more accomodating.
* Kicked off a "your-first-pr" initiative and as a result someone who had never contributed to open source has contributed to the project @ https://github.com/reactiveui/ReactiveUI/issues/1005
* Started evangelism, cold-emailing and as a result ended up in the MSDN .NET blog last week as package of the week.
* Documentation has been fleshed out further and new ToC reorganized @ https://docs.reactiveui.net/en/index.html still sections missing but it's a start. Always looking for help. 
* Design guidelines have been moved to our documentation.
* Over the weekend, work started on production official samples @ https://github.com/reactiveui/samples
* Vector version of our logo has been created @ https://github.com/reactiveui/styleguide

The open source community treats maintainers as a commodity, one of the things we can do to counter-act that is to encourage more people to join our community and eventually become a maintainer if they so choose. You should join us in Slack @ https://github.com/reactiveui/reactiveui#slack
